lib/kernel/test/application_SUITE.erl

```diff
@@ -1037,6 +1037,59 @@ ensure_all_started(_Conf) ->
     ok = application:unload(app_chain_error),
     ok.
 
+optional_applications(_Conf) ->
+    {ok, Fd10} = file:open("app10.app", [write]),
```

Comment:
I don't know why, but for some reason I think this testcase is interacting with the code_SUITE:big_boot_embedded and making it fail. The below is the log form big_boot_embedded.

```
=== Started at 2020-10-27 02:35:10



app10: File not found: "app10.app"



*** CT Error Notification 2020-10-27 02:35:11.032 ***
code_SUITE:create_big_boot failed on line 1424
Reason: {badmatch,error}

Full error description and stacktrace

=== WARNING: Attempt to stop a nonexisting slavenode (big_boot_embedded@artemis)
===          Trying to kill it anyway!!!
```

Reply:
I have addressed this one locally. I forgot to unload app10 after a failed application start on the added test. Thanks!